### PR TITLE
#165727294 index page

### PR DESCRIPTION
**Pull request by AnayoOleru**
#### Description
Currently, there is no index page for users that are not registered yet. This P.R adds a homepage feature for an unregistered user.
#### Type of change

- [x] add reuseable navbar
- [x] add hero-view
- [x] add reuseable article card
- [x] add reuseable footer
- [x] add test

#### How Has This Been Tested?

N/A

#### Checklist:

- [x] add reuseable navbar
- [x] add hero-view
- [x] add reuseable article card
- [x] add reuseable footer
- [x] add test

#### PT-ID

[165727294](https://www.pivotaltracker.com/n/projects/2318856)

#### Screenshots
Screenshot 2019-05-02 at 7 49 06 PM
Screenshot 2019-05-02 at 7 49 17 PM

#### Questions:
N/A

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
     <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```
Comment (sojida) [inline public/index.html]: We already have a font-awesome CDN. Please remove this duplicate. Thanks

File: src/assets/stylesheets/index.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.custom-center{
+    text-align: center;
+}
```
Comment (sojida) [inline src/assets/stylesheets/index.scss]: Please fix this EOL

File: src/components/containers/Homepage/homepage.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.article-section-hmp {
+    width: 80%;
+    margin: 10px auto;
+
+    .advert {
+        width: 100%;
+    }
+}
```
Comment (sojida) [inline src/components/containers/Homepage/homepage.scss]: EOL

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (vic3king) [inline src/assets/stylesheets/index.scss]: Please add EOL

File: src/components/shared/Footer/footer.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        margin: 20px auto;
+        text-align: center;
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 14px;
+        line-height: 23px;
+        color: #959595
+      }
+}
```
Comment (sojida) [inline src/components/shared/Footer/footer.scss]: Please fix this EOL

File: src/components/containers/Homepage/homepage.scss (hunk 3 above)
Comment (vic3king) [inline src/components/containers/Homepage/homepage.scss]: EOL here too.

File: src/components/shared/NavBar/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    .navbar-item:hover, .ui.menu .active.item:hover, .active.item {
+      color: #FFF !important;
+      background-color: #0E71EB !important;
+      transition: 0.5s;
+    }
+    
+    .active.item:hover {
+      background-color: #2185d0 !important;
+    }
+  // }
```
Comment (vic3king) [inline src/components/shared/NavBar/navbar.scss]: if this code block is not in use, we can remove it.

File: src/components/shared/NavBar/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  // img.ui.avatar.image {
+  //   width: 49px;
+  //   height: 58px;
```
Comment (vic3king) [inline src/components/shared/NavBar/navbar.scss]: if this code block is not in use, we can remove it.

**Pull request merged**